modules/oauth-api/src/main/java/org/mule/runtime/oauth/api/PlatformManagedOAuthDancer.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.oauth.api;
+
+import org.mule.api.annotation.Experimental;
+import org.mule.api.annotation.NoImplement;
+import org.mule.runtime.oauth.api.builder.OAuthPlatformManagedDancerBuilder;
+import org.mule.runtime.oauth.api.listener.PlatformManagedOAuthStateListener;
+import org.mule.runtime.oauth.api.state.ResourceOwnerOAuthContext;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Allows to manipulate access and refresh tokens that are obtained and managed by the Anypoint Platform. Each instance is
+ * preconfigured by an {@link OAuthPlatformManagedDancerBuilder} to point to a specific connection id.
+ * <p>
+ * Since the authorizations are performed by the platform, this dancer remains agnostic of the grant type that was used to
+ * obtain them.
+ * <p>
+ * Platform Managed OAuth is an experimental feature. It will only be enabled on selected environments and scenarios.
+ * Backwards compatibility is not guaranteed.
+ *
+ * @since 4.3.0
+ */
+@NoImplement
```

Comment:
won't this implementation be in the service artifact?

Reply:
Yes, on the EE one.